sql/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -529,7 +529,7 @@ object SparkConnectService extends Logging {
 case class SparkListenerConnectServiceStarted(
     hostAddress: String,
     bindingPort: Int,
-    sparkConf: SparkConf,
+    sparkConf: Map[String, String],
```

Comment:
Can anyone explain why we are sending all spark confs? That seems to be a bit heavyweight.

Reply:
regarding use case mentioned in SPARK-47952, I think sparkConf is not necessary, maybe the author can answer this question @TakawaAkirayo

Reply:
The purpose is to send additional context, such as the spark.app.id set in SparkContext's _conf during startup, to the listener. This can be achieved through other means within the listener. I agree that we can remove this from the event.

Reply:
Okay, updated and verified again, please take another look, @hvanhovell and @TakawaAkirayo